.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,6 +7,11 @@ assignees: ''
 
 ---
 
+<!-- NOTE: The Taskcluster team uses Bugzilla as its primary bug-tracking tool.
```

Comment:
Probably "if this issue is not a UI bug or is more than a quick fix". At least to my eye, the current way accidentally implies that UI bugs are simple.

Reply:
Yeah, def not what I meant :)